server/graphql/core/src/generic_filters.rs

```diff
@@ -7,6 +7,8 @@ use repository::{DateFilter, DatetimeFilter, EqualFilter, SimpleStringFilter};
 pub struct SimpleStringFilterInput {
     /// Search term must be an exact match (case sensitive)
     equal_to: Option<String>,
+    /// Search term must be an exact match, but case insensitive
+    insensitive_equal_to: Option<String>,
```

Comment:
Good question. Just had a look at SimpleStringFilter and StringFilter. Can't they be merged? SimpleStringFilter would make sense to me if you only have equal_to or not_equal_to fields, i.e. filter options that are "cheap" so that you can disallow queries that doesn't make sense, e.g. an expensive `like` query on an uuid column. However, there is already `like` in `SimpleStringFilter`. Or was there another motivation for it?

Reply:
I am not sure, we had SimpleStringFilter and EqualFilter<String> in develop, looks like in program there is now StringFilter (used in document.name) and `insensitive_equal_to` added to SimpleStringFilter.

`insensitive_equal_to` actually the same as `like` except it replaces `%` with `""`, and looks like only used once in service, and never used in front end, so I got really confused why `like` is not used (it's not used in front end so why remove `%` or have it ¯\_(ツ)_/¯ ) ?

>  you can disallow queries that doesn't make sense

I made an issue to consolidate all three filters (there is some argument there about why I think we should consolidate all three filters): 

https://github.com/openmsupply/open-msupply/issues/1932

Reply:
Changed to consolidate just the two filters